docs/index.asciidoc

```diff
@@ -57,6 +55,74 @@ Otherwise, you are responsible for maintaining compliance.
 The Logstash open source distribution uses the MaxMind Creative Commons license
 database by default.
 
+==== Database Auto-update
+
+This plugin embedded with Creative Commons (CC) license databases.
+When Logstash works in air-gapped enviornment, CC license databases can be used indefinitely. 
+Logstash checks database update every day. It downloads the latest and can replace the old database on-the-fly.
+Once it downloads EULA license databases, it will not fallback to CC license databases.
+
+If Logstash fails to download the database for 30 days, 
+the geoip filter will stop enriching events in order to maintain compliance.
+Events will be tagged with `_geoip_expired_database` tag to facilitate the handling of this situation.
+
+NOTE: We recommend you allow Logstash to access internet to download databases
+
+==== Database Metrics
+
+You can monitor database status through https://www.elastic.co/guide/en/logstash/current/node-stats-api.html#node-stats-api[Node Stats API]
```

Comment:
please correct me on how to link the plugin doc to logstash doc

Reply:
```suggestion
You can monitor database status through {logstash-ref}/node-stats-api.html#node-stats-api[Node Stats API]
```